### DISCUSSION TITLE
寻求slanet-plus 或slanet 的静态图模型

### DISCUSSION THREAD

**Discussion by shuchao97** [PaddlePaddle/PaddleOCR#14252] [Q&A] [answered]
大佬你好，我最近在测试表格结构识别slanet-plus和slanet 两个的效果都比较好，我目前想将这两个模型转化为rknn，部署到实际的板子上看看效果，但是目前我遇到的问题是这两个模型转换成onnx存在loop算子,动态图导致rknn转换失败。这边有什么可行的解决方案吗

**Comment by GreatV** [comment]
试试最新版的paddle2onnx

**Reply by shuchao97** [reply]
paddle2onnx用的最新的1.3.0 还是会存在一样的问题
![image](https://github.com/user-attachments/assets/cc4cd96a-7950-4230-a266-78234b637cea)
下面这个是我在用这个onnx模型转换rknn出现的问题
![Uploading image.png…]()

**Comment by Zheng-Bicheng** [comment, marked as answer]
RKNN不支持Loop，需要把Loop裁剪掉，手动编写后处理

**Reply by shuchao97** [reply]
我看loop这个地方涉及具体的计算的，裁剪loop后对模型效果因该有影响。因为我对这个模型不太了解，减掉loop后输出节点因该全都变了，这边可以提供这个模型的源码吗。我们这边在测试各种表格识别的模型效果，效果ok的后期可能也会考虑购买

**Reply by GreatV** [reply]
slanet-plus和slanet都是开源的，模型结构可参考https://github.com/PaddlePaddle/PaddleX/blob/bae052d0407641c40429dbeacb89519c5e683fc3/paddlex/repo_apis/PaddleOCR_api/configs/SLANet_plus.yaml

**Reply by Zheng-Bicheng** [reply]
东西都是开源的，不需要付费哈，源码可以看一下瑞芯微电子的 RKNNModelZOO，后处理应该都是一致的

**Reply by Zheng-Bicheng** [reply]
https://github.com/airockchip/rknn_model_zoo

**Comment by kerry-weic** [comment]
@GreatV  [slanet-plus](https://github.com/PaddlePaddle/PaddleX/blob/release/3.0-beta2/paddlex/repo_apis/PaddleOCR_api/configs/SLANet_plus.yaml)和[slanet](https://github.com/PaddlePaddle/PaddleX/blob/release/3.0-beta2/paddlex/repo_apis/PaddleOCR_api/configs/SLANet.yaml)模型结构完全一致？
